src/command.ts

```diff
@@ -5,6 +5,7 @@ import * as Config from '@oclif/config'
 import * as Errors from '@oclif/errors'
 import * as Parser from '@oclif/parser'
 import Help from '@oclif/plugin-help'
+import {sortBy, uniqBy} from '@oclif/plugin-help/lib/util'
```

Comment:
can we import these directly? I'd like to make the help plugin not a strict requirement if possible